modules/cert-manager-query-metrics.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-monitoring.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-query-metrics_{context}"]
+= Querying metrics for the {cert-manager-operator}
```

Comment:
This can also be changed from `the {cert-manager-operator}` to just `cert-manager`.